appinventor/appengine/src/com/google/appinventor/server/DownloadServlet.java

```diff
@@ -244,7 +244,9 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOExc
 
     // Set http response information
     resp.setStatus(HttpServletResponse.SC_OK);
-    resp.setHeader("content-disposition", "attachment; filename=\"" + fileName + "\"");
+    resp.setHeader(
+        "content-disposition", 
```

Comment:
There is extra trailing whitespace on this line.

Reply:
Fixed 👍